src/loggerBuilder.ts

```diff
@@ -0,0 +1,46 @@
+import { createLogger, Logger } from 'winston';
+import Transport from 'winston-transport';
+
+class SimpleConsole extends Transport {
+    log(info: any, callback: () => void) {
+        setImmediate(() => this.emit('logged', info));
+        const msg = [info.meta, info.message];
+        if (info[Symbol.for('splat')]) {
+            msg.push(...info[Symbol.for('splat')]);
+        }
```

Comment:
Is `splat` needed?

Reply:
Yes. Splat is automatically added by Winston if there's more than one arguments given to the logger, the first argument would be under message, and all subsequent arguments would be added to a list under splat. For example, if we say `logger.info('test', 'test2')`, the info object would look like `{message: 'test', level: 'info', Symbol('splat'): ['test2']}`.

Reply:
I thought that this was only needed because we were using `format.splat()`
https://github.com/awslabs/fhir-works-on-aws-interface/pull/65/commits/3baa9fd9e8add990f85d78a382b3ebac93b6488a#diff-e2e506e7052fec29b75b3fab42a462d500bd3321a1a9d2350c1bc822342effd1L48

can you double check?

Reply:
Yes, `Symbol(Splat)` is automatically added with or without `format.splat()`, here's a screenshot of the info object printed without any formatting: 
![Screen Shot 2021-05-06 at 8 17 45 PM](https://user-images.githubusercontent.com/14276357/117380733-63fa0300-aea8-11eb-9856-eb7d8c52f274.png)